codescan/schema.go

```diff
@@ -363,6 +363,10 @@ func (s *schemaBuilder) buildFromType(tpe types.Type, tgt swaggerTypable) error
 			return s.buildFromType(titpe.Underlying(), tgt)
 		}
 
+		if titpe.TypeArgs() != nil && titpe.TypeArgs().Len() > 0 {
+			return s.buildFromType(titpe.Underlying(), tgt)
+		}
```

Comment:
Would be interesting to try to make generation work in a similar way to the example from #3139

And maybe make it work both ways, but I don't have the time to look into it now